server/src/main/resources/org/cloudfoundry/identity/uaa/db/mysql/V4_112__Users_id_to_Varchar.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE users MODIFY id VARCHAR(36) NOT NULL;
```

Comment:
@fhanik can you help @Kehrlann to create dummy migrations, e.g. https://github.com/cloudfoundry/uaa/blob/develop/server/src/main/resources/org/cloudfoundry/identity/uaa/db/postgresql/V4_99_1561608282__NOOP.sql for postgresql and hsql to have same last migration number available

Reply:
Done, thanks!